protocol/message_persistence.go

```diff
@@ -1551,6 +1551,18 @@ func (db sqlitePersistence) SaveMessages(messages []*common.Message) (err error)
 		}
 
 		if msg.ContentType == protobuf.ChatMessage_BRIDGE_MESSAGE {
+			// check updates first
+			var hasMessage bool
+			hasMessage, err = db.bridgeMessageExists(tx, msg.GetBridgeMessage().MessageID)
+			if err != nil {
+				return
```

Comment:
How about we add error log, to be able to instrument code in the future ?

```go
...
log.Errorf("Failed to check if bridge message exists: %v", err)
...
```

Reply:
Do not worry. This function returns (err error). It means that if we just do "return", the err value will be returned. The error is handled in the caller.